### PR TITLE
[Hacktoberfest] add error handlers

### PR DESCRIPTION
**Pull request by Fr3ud**

### CODE REVIEW

File: JavaScript/1-tcp-server.js [hunk 1]
```diff
@@ -2,11 +2,20 @@
 
 const net = require('net');
 
+const onData = data => {
```
Comment (tshemsedinov) [inline JavaScript/1-tcp-server.js]:
```suggestion
const onData = (data) => {
```

File: JavaScript/1-tcp-server.js [hunk 2]
```diff
@@ -2,11 +2,20 @@
...
+  console.log('📨:', data);
+};
+
 net.createServer(socket => {
```
Comment (tshemsedinov) [inline JavaScript/1-tcp-server.js]:
```suggestion
net.createServer((socket) => {
```

**Review by tshemsedinov — APPROVED** [review]
Please add separate commit with obligatory `()` for arrow functions

**Comment by Fr3ud** [comment]
I have added parentheses for the arrow functions

**Pull request merged**